app/controllers/items_controller.rb

```diff
@@ -0,0 +1,74 @@
+class ItemsController < ApplicationController
```

Comment:
__[reek]__
* [InstanceVariableAssumption] assumes too much for instance variable @item [:ref](https://github.com/troessner/reek/blob/master/docs/Instance-Variable-Assumption.md)
* [IrresponsibleModule] has no descriptive comment [:ref](https://github.com/troessner/reek/blob/master/docs/Irresponsible-Module.md)

<!-- Comment by [SideCI](https://sideci.com) -->